.github/pull_request_template.md

```diff
@@ -2,16 +2,24 @@
 
 Version(s):
 <!--- Specify the version or versions of OpenShift your PR applies to.
-If your changes apply to the latest release and/or in-development version of OpenShift, open your PR against the `main` branch. --->
+If your changes apply to the latest release and/or in-development version of OpenShift, open your PR against the `main` branch.
```

Comment:
With the new additions, I think the note about opening the PR against `main` is out of place and should go before the Versions(s) block.